### PR TITLE
added title and description to rendering, added padding on edit screen, ...

### PR DESCRIPTION
**Pull request by banderon**
...updated example code to prevent potential error

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
 === Voce Meta Post Meta Widgets ===
-Contributors: matstars, markparolisi, voceplatforms
+Contributors: matstars, markparolisi, banderon, voceplatforms
 Tags: post, meta, widgets, widget area
 Requires at least: 3.3
 Tested up to: 3.9.0
```
Comment (matgargano) [inline readme.txt:5]: Let's test on the latest version of WP 3.9.1

Comment (banderon) [inline readme.txt:5]: Alright.. let me know if you notice any problems :+1:

File: readme.txt [hunk 2]
```diff
@@ -37,11 +37,14 @@ add_filter('voce_post_meta_widgets_post_types', function( $current_post_types )
 ```
 1. Display the sidebar on a page
 ```
-if( !dynamic_sidebar( Voce_Post_Meta_Widgets::WIDGET_ID_PREFIX . $group_name . '_' . $field_name . '_post_id_' . get_the_ID() )): endif;
+if( class_exists( 'Voce_Post_Meta_Widgets' ) && !dynamic_sidebar( Voce_Post_Meta_Widgets::WIDGET_ID_PREFIX . $group_name . '_' . $field_name . '_post_id_' . get_the_ID() )): endif;
 ```
 
 == Changelog ==
 
+= 0.5 =
```
Comment (kevinlangleyjr) [inline readme.txt]: Did you mean 0.2 here?

Comment (banderon) [inline readme.txt]: I was reaching towards the future.

**Pull request merged**